lib/tasks/mutant.rake

```diff
@@ -0,0 +1,17 @@
+CLASSES = %w[
+  CaseRequest
+].freeze
+
+task :mutant do
+  CLASSES.each do |klass|
+    vars = 'RAILS_ENV=test NOCOVERAGE=true'
+    flags = '--use rspec --fail-fast'
+    unless system("#{vars} mutant -r ./config/environment #{flags} #{klass}")
```

Comment:
@tatyree Cool. If you are interested to talk about your challenge in detail feel free to join the mutaiton testing slack channel, link in mutants readme.